<launch>
	<group ns="vision">
		<node name="facenet" pkg="facenet_ros" type="facenet_node.py" output="screen">
			<param name="model_file" value="$(env HOME)/docker_volumen/facenet_models/20180402-114759/" />
			<param name="classifier_file" value="$(env HOME)/docker_volumen/facenet_models/biorobotica_classifier.pkl" />
			<param name="training_dir" value="$(env HOME)/docker_volumen/facenet_datasets/biorobotica/biorobotica_mtcnnpy_160"/>
			<param name="classifier_mode" value="False"/>
			<param name="batch_size" value="5"/>
			<param name="image_size" value="160"/>
			<param name="margin" value="32"/>
			<param name="gpu_memory_fraction" value="0.2"/>
			<param name="detect_multiple_faces" value="True"/>
			<param name="threshold_reco" value="0.0"/>
		</node>
	</group>
</launch>
